src/pk-main.c

```diff
@@ -32,6 +32,7 @@
 #include <glib-unix.h>
 #include <glib/gi18n.h>
 #include <packagekit-glib2/pk-debug.h>
+#include <systemd/sd-daemon.h>
```

Comment:
I think you'll need to add something like HAVE_SYSTEMD_SD_DAEMON_H around this otherwise you'll upset the usual crowd.